redi-connect-backend/package.json

```diff
@@ -6,8 +6,8 @@
     "node": ">=6"
   },
   "scripts": {
-    "start": "node .",
-    "start:dev": "NODE_ENV=development HTTP=1 node .",
+    "start": "NODE_ENV=development node .",
+    "start:prod": "NODE_ENV=prod node .",
```

Comment:
Same comment here.... I would much prefer to be overly explicit here about what `prod` actually means.

Reply:
this is indeed right, I ve missed to revert this bit

Reply:
Would you prefer to have this named production? For me is only a way to run the the backend in the Dockerfile

Reply:
Yep definitely agree on that, not huge fun of abbreviations either. Will change the script to be `start:production` makes sense, thanks for bringing this up Eric!